src/winetricks

```diff
@@ -942,6 +942,14 @@ w_download_to()
                ${_W_cookiejar:+--cookie "$_W_cookiejar"} \
                ${_W_agent:+--user-agent "$_W_agent"} \
                "$_W_url"
+        elif [ "${WINETRICKS_DOWNLOADER}" = "fetch" ]
+        then
+           # shellcheck disable=SC2086
+           $torify fetch \
+               -T "${WINETRICKS_DOWNLOADER_TIMEOUT}" \
```

Comment:
Does fetch work under torify (if torify available on FreeBSD?)

This also lacks support for $WINETRICKS_DOWNLOADER_RETRIES